website/docs/theme-classic.md

```diff
@@ -57,6 +57,7 @@ module.exports = {
       logo: {
         alt: 'Site Logo',
         src: 'img/logo.svg',
+        darkSrc: 'img/logo_dark.svg', // default to logo.src
```

Comment:
Maybe is it better to rename to `srcDark` or just `dark`?

Reply:
I think `srcDark` would be better.